extensions/wikia/PageHeader/PageHeaderController.class.php

```diff
@@ -16,4 +22,13 @@ public function categories() {
 	public function subtitle() {
 		$this->setVal( 'subtitle', new Subtitle( $this->app ) );
 	}
+
+	public function actionButton() {
+		global $wgTitle;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)